src/documentation/markdown/2019/lar-data-fields.md

```diff
@@ -400,7 +400,7 @@
   - 48%
   - 49%
   - 50%-60%
-  - >60%
+  - 60%
```

Comment:
Correct bin label?

Reply:
I don't recall why this value was changed. I will bring up to the team during standup.

Reply:
Has been changed back to `>60%` via standup feedback.